src/main/java/sirius/db/mixing/EntityDescriptor.java

```diff
@@ -215,6 +219,39 @@ protected EntityDescriptor(Class<?> type) {
         return type;
     }
 
+    /**
+     * Builds a standard message that the given entity cannot be saved.
+     * <p>
+     * This message is mostly used as a consequence describing why an error happened.
+     * For instance, when an entity's required field is not populated, a standard message like this will be logged:
+     * {@code Field 'MyField' is required.}
+     * As to would be rather nice to tell the end user that an entity couldn't be saved due to the error.
+     * {@code MyEntity cannot be saved: Field 'MyField' is required.}
+     *
+     * @param clazz the entity class to extract its label
+     * @return the formatted message
+     */
+    public static String createCannotSaveMessage(@Nonnull Class<?> clazz) {
+        return NLS.fmtr("EntityDescriptor.cannotSaveEntity")
+                  .set("entity", mixing.getDescriptor(clazz).getLabel())
+                  .format();
+    }
+
+    /**
+     * Builds a standard message that the given entity cannot be saved.
+     * <p>
+     * This message is mostly used as a consequence describing why an error happened.
+     * For instance, when an entity's required field is not populated, a standard message like this will be logged:
+     * {@code Field 'MyField' is required.}
+     * As to would be rather nice to tell the end user that an entity couldn't be saved due to the error.
+     * {@code MyEntity cannot be saved: Field 'MyField' is required.}
```

Comment:
```suggestion
     * This message is mostly used as a consequence for describing why an error happened. Therefore, this will 
     * create message like <tt>Enttiy cannot be saved: Field-Error</tt> instead of logging an abitrary looking field 
     * error alone.
```